index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>MEdicore</title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="section--pink">
```

Comment:
In BEM modifiers are used in combination with block or element classes. `section--pink` looks like a modifier. If it is, perhaps say `class="section section--pink"` If it is a simple section then `class="section"` or `class="section-pink"` would do.

Reply:
Ibahim noticed this too! thanks.
resolved 👍